go.mod

```diff
@@ -23,11 +23,11 @@ require (
 	github.com/prometheus/common v0.44.0 // indirect
 	github.com/prometheus/procfs v0.10.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/net v0.12.0 // indirect
+	golang.org/x/net v0.13.0 // indirect
 	golang.org/x/sys v0.10.0 // indirect
 	golang.org/x/text v0.11.0 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20230525234030-28d5490b6b19 // indirect
 	google.golang.org/protobuf v1.30.0 // indirect
-	k8s.io/apimachinery v0.28.0-rc.0 // indirect
+	k8s.io/apimachinery v0.28.0 // indirect
 	k8s.io/component-base v0.28.0-rc.0 // indirect
```

Comment:
Component-base v0.28.0 is out https://github.com/kubernetes/component-base/tree/v0.28.0

Reply:
This is coming from [csi-lib-utils](https://github.com/kubernetes-csi/csi-lib-utils) which is on version `v0.15.0-rc0`. We will have to merge csi-lib-utils first to bump this to `v0.28.0`

Reply:
I though we cannot update the "indirect" versions directly, turns out you can. I have bumped the component-base version